client/guidestours/style.scss

```diff
@@ -30,7 +30,51 @@
 	}
 }
 
-// variables for the guidestours__bullseye
+.guidestours__choice-button-row {
+	button {
+		width: 48%;
+	}
+	button:nth-child(1) {
+		margin-right: 4%;
+	}
+	button.guidestours__secondary-button {
+		background: $gray-light;
+		color: darken( $gray, 20% );
+	}
+}
+
+.guidestours__single-button-row {
+	button {
+		width: 100%;
+	}
+}
+
+.guidestours__external-link,
+.guidestours__bullseye-instructions {
+	p {
+		color: darken( $gray, 10 );
+		margin-bottom: 0;
+		font-style: italic;
+	}
+
+	.gridicon {
+		position: relative;
+		top: 7px;
+	}
+
+	.external-link {
+		border-top: 1px solid $gray-light;
+		display: block;
+		padding-top: 8px;
+		margin-top: 16px;
+	}
+}
+
+.guidestours__bullseye-instructions {
+	margin-top: -7px;
+}
+
+// the bullseye used for showing an action step's target
```

Comment:
Did a rebase this morning, no dice. Looking again now I realize of course it looks like this: I resorted the CSS in 562eb78. All good.